.github/workflows/npm.yml

```diff
@@ -58,14 +58,14 @@ jobs:
     - run: yarn install --ignore-scripts --frozen-lockfile
       working-directory: node
 
-    - run: |
-        if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
-          sudo apt-get install -y crossbuild-essential-arm64
-          export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc
-        fi
-        npx prebuildify --napi -t ${{ steps.get-nvm-version.outputs.node-version }} --arch arm64
+    - name: Install crossbuild for Linux arm64
+      run: |
+        sudo apt-get install -y crossbuild-essential-arm64
+        echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
```

Comment:
Hmm, "This file is unique to the current step and changes for each step in a job." But we could just add this environment variable unconditionally to the next step, it's only going to affect arm64 cross-compiling and doesn't have any paths in it.

Reply:
Haha yeah I also got a bit confused by the docs - tested it in my example run and the env variable was indeed added to all next steps after it was set :)